app/views/collections/index.html.erb

```diff
@@ -22,27 +22,30 @@
 
 <!--Subnavigation, sorting and actions-->
 <h3 class="landmark heading"><%= ts("Navigation") %></h3>
-<ul class="navigation actions" role="navigation">
-  <% # Collections and Open Challenges links unless a logged in user viewing own collections page %>
+<ul class="navigation actions">
+# Collections and link unless a logged in user viewing own collections page %>
   <% unless logged_in? && @user && @user == current_user %>
     <li><%= span_if_current ts("Collections"), collections_path %></li>
-    <li><%= link_to ts("Open Challenges"), list_challenges_collections_path %></li>
+  <% end %>
+# Open Challenges link unless a logged in user viewing own collections page or on an individual collection page %>
+  <li><%= link_to ts("Open Challenges"), list_challenges_collections_path %></li>
+  <% unless (logged_in? && @user && @user == current_user) || @collection %>
```

Comment:
I think these two lines got switched. The link should be inside the conditional.

The conditional should also be `<% unless @user || @collection %>`, since we never want the Open Challenges link on a user's collections page. I think when I wrote the issue, I only checked my own collections page and foolishly assumed the link would also be omitted if I looked at another user's page.